layouts/partials/navigators/navbar.html

```diff
@@ -31,16 +31,7 @@
     <div class="collapse navbar-collapse" id="top-nav-items">
       <ul class="navbar-nav ml-auto">
         <li class="nav-item">
-          <a
-            class="nav-link"
-            href="#home"
-            data-toggle="collapse"
-            data-target=".navbar-collapse"
-            aria-controls="collapseExample"
-            role="button"
```

Comment:
Breaking down the example more https://getbootstrap.com/docs/4.0/components/navbar/ they didn't use these for the actual links, these were for the item that toggled a drop down vs we just have a drop down when in mobile, so i'm removing these